### ISSUE TITLE
ERROR: input directory 'abg' does not exist while recompiling

### ISSUE THREAD

**Issue by Bonjur** [iBotPeaches/Apktool#1351]
### Information
1. 2.0.3
2. Windows 8.1
3. Playstore
### Stacktrace/Logcat

```
ERROR: input directory 'abg' does not exist
Exception in thread "main" brut.androlib.AndrolibException: brut.androlib.AndrolibException: brut.common.BrutException: could not exec command: [C:\Users\dragon                                                   \AppData\Local\Temp\brut_util_Jar_6494125442485072350.tmp, p, --forced-package-id, 127, --min-sdk-version, 16, --target-sdk-version, 24, --version-code, 42, --version-name, 1.8.23, -F, C:\Users\dragon                                                       \AppData\Local\Temp\APKTOOL4026322081030667743.tmp, -0, arsc, -0, json, -0, txt, -0, , -0, abg, -0, version, -0, arsc, -I, E:\p.apk\AndroidManifest.xml]
    at brut.androlib.Androlib.buildResourcesFull(Androlib.java:472)
    at brut.androlib.Androlib.buildResources(Androlib.java:410)
    at brut.androlib.Androlib.build(Androlib.java:298)
    at brut.androlib.Androlib.build(Androlib.java:268)
    at brut.apktool.Main.cmdBuild(Main.java:225)
    at brut.apktool.Main.main(Main.java:84)
Caused by: brut.androlib.AndrolibException: brut.common.BrutException: could not exec command: [C:\Users\dragon                                                       \AppData\Local\Temp\brut_util_Jar_6494125442485072350.tmp, p, --forced-package-id, 127, --min-sdk-version, 16, --target-sdk-version, 24, --version-code, 42, --version-name, 1.8.23, -F, C:\Users\dragon                                                       \AppData\Local\Temp\APKTOOL4026322081030667743.tmp, -0, arsc, -0, json, -0, txt, -0, , -0, abg, -0, version, -0, arsc, -I, E:\Frameworks\1.apk, -S, E:\p.apk\res, -M, E:\p.apk\AndroidManifest.xml]
    at brut.androlib.res.AndrolibResources.aaptPackage(AndrolibResources.java:425)
    at brut.androlib.Androlib.buildResourcesFull(Androlib.java:458)
    ... 5 more
Caused by: brut.common.BrutException: could not exec command: [C:\Users\dragon                                                       \AppData\Local\Temp\brut_util_Jar_6494125442485072350.tmp, p, --forced-package-id, 127, --min-sdk-version, 16, --target-sdk-version, 24, --version-code, 42, --version-name, 1.8.23, -F, C:\Users\dragon                                                       \AppData\Local\Temp\APKTOOL4026322081030667743.tmp, -0, arsc, -0, json, -0, txt, -0, , -0, abg, -0, version, -0, arsc, -I, E:\1.apk, -S, E:\p.apk\res, -M, E:\p.apk\AndroidManifest.xml]
    at brut.util.OS.exec(OS.java:89)
    at brut.androlib.res.AndrolibResources.aaptPackage(AndrolibResources.java:419)
    ... 6 more
```
### APKTOOL.YML

version: 2.0.3
apkFileName: 
isFrameworkApk: false
usesFramework:
  ids:
- 1
  sdkInfo:
  minSdkVersion: '16'
  targetSdkVersion: '24'
  packageInfo:
  forced-package-id: '127'
  versionInfo:
  versionCode: '42'
  versionName: 1.8.23
  sharedLibrary: false
  doNotCompress:
  - arsc
  - json
  - txt
  - '' (TRIED REMOVING '' BUT STILL SAME ERROR)
  - abg
  - version

**Comment by iBotPeaches** [comment]
Do you have an apk? It seems apktool detected an unknown file with the `abg` extension, but during compilation it was not found. It this is a detection bug, I want to patch this.

If you removed the unknown file after decode, but before compile. I can probably patch that as well into a warning vs dying out.

**Comment by Bonjur** [comment]
This is the APK, decompile it and try to recompile you'll get the error: https://apkpure.com/plants-vs-zombies%E2%84%A2-heroes/com.ea.gp.pvzheroes

**Comment by Bonjur** [comment]
@iBotPeaches

**Comment by iBotPeaches** [comment]
Sorry for delay. This was fixed way back in 2.1.0 with this commit - https://github.com/iBotPeaches/Apktool/issues/1122

As you can see with the latest version.

```
➜  Bug1351 apktool d Plants\ vs\ Zombies™\ Heroes_v1.8.26_apkpure.com.apk 
I: Using Apktool 2.2.2-7c6c7a-SNAPSHOT on Plants vs Zombies™ Heroes_v1.8.26_apkpure.com.apk
I: Loading resource table...
I: Decoding AndroidManifest.xml with resources...
I: Loading resource table from file: /home/ibotpeaches/.local/share/apktool/framework/1.apk
I: Regular manifest package...
I: Decoding file-resources...
I: Decoding values */* XMLs...
I: Baksmaling classes.dex...
I: Copying assets and libs...
I: Copying unknown files...
I: Copying original files...
➜  Bug1351 apktool b Plants\ vs\ Zombies™\ Heroes_v1.8.26_apkpure.com
I: Using Apktool 2.2.2-7c6c7a-SNAPSHOT
I: Checking whether sources has changed...
I: Smaling smali folder into classes.dex...
I: Checking whether resources has changed...
I: Building resources...
I: Copying libs... (/lib)
I: Building apk file...
I: Copying unknown files/dir...
➜  Bug1351 
```
